src/lib/list/selection-list.spec.ts

```diff
@@ -232,6 +232,52 @@ describe('MdSelectionList', () => {
     });
   });
 
+  describe('with option disabled', () => {
+    let fixture: ComponentFixture<SelectionListWithDisabledOption>;
+    let listOptionEl: HTMLElement;
+    let listOption: MdListOption;
+
+    beforeEach(async(() => {
```

Comment:
Why are there two separate `beforeEach`?

Reply:
Because otherwise we'd need to chain into `TestBed.compileComponents()`. That's basically how we do it in every test file.